.github/CODEOWNERS

```diff
@@ -8,3 +8,6 @@
 
 # The java-samples-reviewers team is the default owner for samples changes
 samples/**/*.java       @googleapis/java-samples-reviewers
```

Comment:
File error, most likely due to java-samples-reviewers not being a contributor to the repo

Reply:
Added java-samples-reviewers as Write role